quantumdb-postgresql/pom.xml

```diff
@@ -4,11 +4,24 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
+	<build>
```

Comment:
This block isn't needed as the compiler plugin basically ships by default, and uses the properties in the parent POM: https://github.com/quantumdb/quantumdb/blob/master/pom.xml#L22-L23

Reply:
Ah ok, Maven is still sort of a mystery for me on how to correctly configure it.